doc/nrf/ug_tfm.rst

```diff
@@ -101,12 +101,29 @@ Logging
 TF-M employs two UART interfaces for logging: one for the secure part (MCUboot and TF-M), and one for the non-secure application.
 The logs arrive on different COM ports on the host PC.
 
-.. note::
-   * On the nRF5340 DK v1.0.0, you must connect specific wires on the kit to receive secure logs on the host PC.
-     Specifically, wire the pins **P0.25** and **P0.26** of the **P2** connector respectively to **RxD** and **TxD**  of the **P24** connector.
-     See :ref:`logging_cpunet` on the Working with nRF5340 DK page for more information.
-   * On the nRF5340 DK v2.0.0, there is one fewer COM port than on v1.0.0, so the secure and non-secure UART peripheral must be wired to the same pins.
-     Specifically, wire the pins **P0.25** and **P0.26** to **P0.20** and **P0.22**, respectively.
+
+Virtual COM ports in nRF5340 DK
+===============================
+On the nRF5340 DK v1.0.0, you must connect specific wires on the kit to receive secure logs on the host PC.
+Specifically, wire the pins **P0.25** and **P0.26** of the **P2** connector respectively to **RxD** and **TxD**  of the **P24** connector.
+See :ref:`logging_cpunet` on the Working with nRF5340 DK page for more information.
+
+On the nRF5340 DK v2.0.0, there are only two virtual COM ports available.
+By default, one of the ports is used by the non-secure UART peripheral from the application and the other one by the UART1 peripheral from the network core.
+
+There are several options to get UART output from the secure TF-M.
+
+* Disable the output for the network core.
+  For NCS samples, see :ref:`ug_multi_image_variables`.
+  The RXD (:kconfig:option:`TFM_UART1_RXD_PIN`) and TXD (:kconfig:option:`TFM_UART1_TXD_PIN`) pins of TF-M secure can be set to **P1.00** (32) and **P1.01** (33).
```

Comment:
Have you followed these intructions and can confirm they work?

Reply:
The instructions seem to only cover how to configure the secure pins, is there anything we can do to help them understand how to disable output on the network core?

Reply:
> Have you followed these instructions and can confirm they work?

Sure tests it on my desk, but happy if somebody wants to test this as well.

Reply:
> The instructions seem to only cover how to configure the secure pins, is there anything we can do to help them understand how to disable output on the network core?

Well that depends on what you run on the network core, that will get a bit out of scope here in my opinion